app/api/resources/user.py

```diff
@@ -129,7 +129,14 @@ def get(cls):
     @users_ns.doc("update_user_profile")
     @users_ns.expect(auth_header_parser, update_user_request_body_model)
     @users_ns.response(HTTPStatus.OK, "%s" % messages.USER_SUCCESSFULLY_UPDATED)
-    @users_ns.response(HTTPStatus.BAD_REQUEST, "%s" % messages.NAME_INPUT_BY_USER_IS_INVALID)
+    @users_ns.response(
+        HTTPStatus.BAD_REQUEST,
+        "%s\n%s"
+        % (
+            messages.NAME_INPUT_BY_USER_IS_INVALID,
+            messages.NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT
+        )
```

Comment:
Can you please use f-strings for this? we have some issues especially for converting these parts of the code to use f-strings :)
```
f"{messages.NAME_INPUT_BY_USER_IS_INVALID}\n"
f"{messages. NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT}\n"
```

Reply:
@isabelcosta please review it once again